README.md

```diff
@@ -313,7 +313,7 @@ receiver: team-X-pager
 Amtool allows you to visualize the routes of your configuration in form of text tree view.
 Also you can use it to test the routing by passing it label set of an alert
 and it prints out all receivers the alert would match ordered and separated by `,`.
-(If you use `--verify.receivers` amtool returns error code 1 on mismatch)
+(If you use `--verify.receivers` amtool returns error code 1 on mismatch.)
```

Comment:
no need for the additional dot

Reply:
You mean the period at the end of the sentence?

Reply:
yes